src/basic/io-util.c

```diff
@@ -254,11 +254,19 @@ ssize_t sparse_write(int fd, const void *p, size_t sz, size_t run_length) {
         return q - (const uint8_t*) p;
 }
 
-char* set_iovec_string_field(struct iovec *iovec, size_t *n_iovec, const char *field, const char *value) {
+char *set_iovec_string_field(struct iovec *iovec, size_t *n_iovec, const char *field, const char *value) {
```

Comment:
I guess to make it similar to variable statements where "char *var;" is nicer. I don't think it's part of the coding style at least I couldn't find relevant info.

Reply:
But in case of a variable, the variable itself is a pointer, so that's OK. But here the function (even though a pointer itself), does not return type, but the pointer to the type.